tests/test_schema.py

```diff
@@ -1150,12 +1194,10 @@ class World(Base):
 
         world_schema = SQLAlchemySchemaNode(World)
 
-        self.assertTrue(
-            isinstance(world_schema['not_a_number'].typ, colander.String))
+        self.assertIsInstance(
```

Comment:
I'd like to embrace the future too, but unfortunately `assertIsInstance` doesn't exist in Py2.6 and we have down that we support that.